modules/OCPPConfiguration/config-mapping.yaml

```diff
@@ -0,0 +1,312 @@
+- component:
+    name: ocpp
+    instance: 1
+    evse:
+      id: 1
+      connector: 1
+  variable:
+    name: AllowOfflineTxForUnknownId
+    instance: 1
+  maps_to:
+    module_id: allow_offline_tx_for_unknown_id
+    config_key: allow_offline_tx_for_unknown_id_config
+- component:
+    name: ocpp
+    instance: 2
+    evse:
+      id: 2
+      connector_id: 2
+  variable:
+    name: AllowOfflineTxForUnknownId
+    instance: 2
+  maps_to:
+    module_id: allow_offline_tx_for_unknown_id
+    config_key: allow_offline_tx_for_unknown_id_config
+- component:
+    name: ocpp
+    instance: 1
+    evse:
+      id: 1
+      connector: 1
+  variable:
+    name: AuthorizationCacheEnabled
+    instance: 1
+  maps_to:
+    module_id: authorization_cache_enabled
+    config_key: authorization_cache_enabled_config
```

Comment:
Please separate the definition of one mapping into:
* one level for ocpp definition
* one level for everest definition

Reply:
Done